OOVPADatabase/D3D8LTCG_OOVPA.inl

```diff
@@ -65,7 +65,7 @@
 // ******************************************************************
 OOVPATable D3D8LTCG_OOVPAV2[] = {
 
-    REGISTER_OOVPAS(D3D_MakeRequestedSpace, 1024, 1036), //TODO: Need to verify, both versions are in 5849. Plus D3D8 symbol is detected.
+    REGISTER_OOVPAS(D3D_MakeRequestedSpace_8, 1036), // NOTE: The verison is in 5849. Being used by (5849) Manhunt title
```

Comment:
- "verison"?
- "is in 5849"?

The whole sentence is messed up (both of them?).

Reply:
Rephased to `OOVPA is in 5849`, I was distracted when I made the change to match @jarupxx's comment.